src/scala/com/github/johnynek/bazel_deps/Target.scala

```diff
@@ -20,18 +27,34 @@ object Target {
     fqn.toLowerCase.replaceAll("[^a-z0-9]", "_")
 
   sealed abstract class Kind(override val toString: String)
+  object Kind {
+    def parse(str: String): Result[Kind] = str match {
+      case "library" => IO.const(Library)
+      case "import" => IO.const(Import)
+      case "test" => IO.const(Test)
+      case "binary" => IO.const(Binary)
```

Comment:
can we make this total: `case other => IO.failed(new IllegalArgumentException(s"unexpected library kind: $other"))`

Reply:
👍